.github/workflows/ci.yml

```
@@ -551,9 +545,72 @@
           fi
           echo "TAG=$TAG" >> $GITHUB_OUTPUT
 
+  release-auto:
+    name: Create Release from Auto-Tag
+    needs: [docker, auto-tag]
+    runs-on: ubuntu-latest
+    # Run when auto-tag creates a new tag on main branch
+    if: |
+      github.ref == 'refs/heads/main' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)